sda-download/dev_utils/run_integration_test.sh

```diff
@@ -194,7 +194,7 @@ echo "expected file found"
 # Test file can be decrypted
 ## test also the files endpoint
 
-C4GH_PASSPHRASE=$(grep -F passphrase config.yaml | sed -e 's/.* //' -e 's/"//g')
+C4GH_PASSPHRASE=$(grep -F passphrase config.yaml | tail -1 | sed -e 's/.* //' -e 's/"//g')
```

Comment:
```suggestion
C4GH_PASSPHRASE=$(grep -F passphrase config.yaml | sed -e 's/.* //' -e 's/"//g')
```
Shouldn't be needed since the config file only will have one cryptgh config block

Reply:
If you want to extract a specific entry using `yq` is the simplest solution.
```sh
yq '.app.c4gh.passphrase' config.yaml
```

Reply:
changed all similar  to using `yq`